maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -0,0 +1,124 @@
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<!--
+   Define Plexus components to allow this plugin to function as
+   a Maven extension that defines new packaging types 'galleon-feature-pack' and 'galleon-user-feature-pack'.
+   When building a module with these types the
+-->
+<component-set>
+  <components>
+    <component>
+      <role>org.apache.maven.lifecycle.mapping.LifecycleMapping</role>
+      <role-hint>galleon-feature-pack</role-hint>
+      <implementation>org.apache.maven.lifecycle.mapping.DefaultLifecycleMapping</implementation>
+      <configuration>
+        <lifecycles>
+          <lifecycle>
+            <id>default</id>
+            <phases>
+              <process-resources>
+                org.apache.maven.plugins:maven-resources-plugin:resources
+              </process-resources>
+              <compile>
+                org.apache.maven.plugins:maven-compiler-plugin:compile
+              </compile>
+              <process-test-resources>
+                org.apache.maven.plugins:maven-resources-plugin:testResources
+              </process-test-resources>
+              <test-compile>
+                org.apache.maven.plugins:maven-compiler-plugin:testCompile
+              </test-compile>
+              <test>
+                org.apache.maven.plugins:maven-surefire-plugin:test
+              </test>
+              <package>
+                <!-- Replace maven-jar-plugin -->
+                org.wildfly.galleon-plugins:wildfly-galleon-maven-plugin:build-feature-pack
+              </package>
+              <install>
+                org.apache.maven.plugins:maven-install-plugin:install
+              </install>
+              <deploy>
+                org.apache.maven.plugins:maven-deploy-plugin:deploy
+              </deploy>
+            </phases>
+          </lifecycle>
+        </lifecycles>
+      </configuration>
+    </component>
+    <component>
+      <role>org.apache.maven.artifact.handler.ArtifactHandler</role>
+      <role-hint>galleon-user-feature-pack</role-hint>
+      <implementation>
+        org.apache.maven.artifact.handler.DefaultArtifactHandler
+      </implementation>
+      <configuration>
+        <!--the extension used by Maven in the repository-->
+        <extension>zip</extension>
+        <!--the type used when specifying dependencies etc.-->
+        <type>zip</type>
+        <!--the packaging used when declaring an implementation of
+          the packaging-->
+        <packaging>galleon-feature-pack</packaging>
```

Comment:
seems not used, should be galleon-user-feature-pack.

Reply:
Actually, it's L54 that should change to 'galleon-feature-pack'.  This is a good sign this component is not needed though. :)

Reply:
Removed